WASM-specs.bib

```diff
@@ -0,0 +1,8 @@
+@report{WebAssemblyCoreSpecification,
+  title = {{{WebAssembly Core Specification}}},
+  editor = {Rossberg, Andreas},
+  date = {2019-12-05},
+  institution = {{W3C}},
+  url = {https://www.w3.org/TR/wasm-core-1/},
```

Comment:
I wonder if we could also point to the PDF somehow, perhaps as a note?

Reply:
Have a look at the renamed file, does that look good?